### PR TITLE
Identify tool - Edit mode

### PR DESCRIPTION
**Pull request by vlt1**
## Description
Adds edit buttons to Identify tool that open Attribute Table

**Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines: https://github.com/geosolutions-it/MapStore2/blob/master/CONTRIBUTING.md
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**What kind of change does this PR introduce?** (check one with "x", remove the others)
 - [ ] Bugfix
 - [ ] Feature
 - [ ] Code style update (formatting, local variables)
 - [ ] Refactoring (no functional changes, no api changes)
 - [ ] Build related changes
 - [ ] CI related changes
 - [x] Other... Please describe: enhancement

## Issue
#5185
[mapstore2-georchestra/#45](https://github.com/georchestra/mapstore2-georchestra/issues/45)

## Breaking change
**Does this PR introduce a breaking change?** (check one with "x", remove the other)
 - [ ] Yes, and I documented them in migration notes
 - [x] No

### CODE REVIEW

**Comment by tdipisa** [comment]
@offtherailz I think the edit button should stay in the top toolbar

**Comment by offtherailz** [comment]
@tdipisa yes, this is the sample of "edit single feature" (that have to be removed) . I provided this example in animation because it was the only one that was filtering by point, in the current PR. 
In global version (present for text/html only format, but it will be only one in the final version) the button is in the right place.

File: web/client/actions/wfsquery.js [hunk 1]
```diff
@@ -139,12 +141,13 @@ function createQuery(searchUrl, filterObj) {
     };
 }
 
-function query(searchUrl, filterObj, queryOptions) {
+function query(searchUrl, filterObj, queryOptions, reason) {
```
Comment (offtherailz) [inline web/client/actions/wfsquery.js:144]: I understand why you need this additional field. Action sequence is not enough to understand if activate or not autoselect? like geometryFilterActive + updateFilter? It should work when you select from the feature grid and when you start from the Feature Info
If it is not, please document well this method and the point you use to intercept this.

File: web/client/actions/featuregrid.js [hunk 2]
```diff
@@ -204,11 +204,11 @@ function setLayer(id) {
...
     };
 }
-function updateFilter(update) {
+function updateFilter(update, dontUpdateQuery) {
     return {
         type: UPDATE_FILTER,
-        update
-
+        update,
+        dontUpdateQuery
```
Comment (offtherailz) [inline web/client/actions/featuregrid.js:211]: Please let me know if we can move this logic  inside the epic and avoid this flag to be passed through components

File: web/client/epics/featuregrid.js [hunk 3]
```diff
@@ -242,29 +258,23 @@ module.exports = {
                         .take(1)
                     )
             ),
+    featureGridUpdateGeometryFilter: (action$, store) =>
```
Comment (offtherailz) [inline web/client/epics/featuregrid.js:261]: I think you should put this logic in the `featureGridUpdateFilter` and/or document it. This way the update logic is all together and documented. Splitting it into different epics decrese readability, if you don't notice that the two filters are mutually exclusive.

**Pull request merged**